geosys/bridge_api/test/test_field_level_maps.py

```diff
@@ -25,7 +26,16 @@ class BridgeAPIFieldLevelMapsTest(unittest.TestCase):
     def setUp(self):
         """Runs before each test."""
         client = ConnectionAPIClientV2(IDENTITY_URLS['na']['test'])
-        json = client.get_access_token('Bridge_US_Demo', 'Welcome12k18')
+        username = os.environ.get('BRIDGE_API_USERNAME', None)
```

Comment:
Here you should be using QSettings because in your plugin you are going to have an options panel where you write the credentials to the plugin ini file. I think in your testing strategy, you will just get the env var at test class constructor, set the QSettings properties and then the rest of code can assume credentials will come from QSettings. Make sense?

Reply:
yes, I was also planning to use QSetting but it will be in the plugin implementation and unittest. this is the API interface implementation so I was thinking to make it simple. CMIIW